### DISCUSSION TITLE
Strange chart bug when update

### DISCUSSION THREAD

**Discussion by Topex22** [poblabs/weewx-belchertown#859] [Q&A]
Hello everyone and best wishes for the new year.

 This is what happens when I upgrade from belchertown.py v1.2 to belchertown.py v1.3

 ![This is an image](http://chevrerie.du.cap.chez-alice.fr/Bel_smooth.jpg)

 ![This is an image](http://chevrerie.du.cap.chez-alice.fr/Bel_scale.jpg)

 I absolutely don't understand why.

 Particularity: this is data that I added manually to the database.

 If you have an idea...

**Comment by Brezentrager** [comment]
Hi Topex22,
I've experienced the same issue.

Increasing the number of decimals in skin.conf / whew.conf did the job in my case.
[Units] [[StringFormats]]

Downside is that the affected measures are now also displayed with more decimals in the rest of the website.

Ideally, we would be able to set the number of decimals independently for the skin and the charts.

**Comment by Topex22** [comment]
Hi Brezentrager,
It does not work.  I added hauteur = %.2f in the [[StringFormats]] of skin.conf and weewx.conf.  I have my two decimas but still the stair effect in steps of 1.00. It's the same with Evapotranspiration

**Reply by Brezentrager** [reply]
You can't associate decimals with a measurement variable (like Evapotranspiration). It's associated with the the measurement unit (like mm).

Have you tried increasing the decimals for the associated unit?

**Comment by Topex22** [comment]
idiot I am !!! 😖

**Reply by fvirgola80** [reply]
Hi @Topex22, I'm very interested in the tide graph, can I ask how I could proceed in order to have the same result as yours?
www.meteocivitavecchia.it
my email is fvirgola80@gmail.com

Thank you

**Comment by Topex22** [comment]


**Reply by fvirgola80** [reply]
Thanks for the reply, I don't have the sea times for my location.... maybe we can write to each other by email, what do you think?
what is the address of the site?

Thank you :-)
